sw/device/tests/BUILD

```diff
@@ -5576,3 +5576,44 @@ opentitan_test(
         "//sw/device/lib/testing/test_framework:ottf_main",
     ],
 )
+
+opentitan_test(
+    name = "uart_parity_test",
+    srcs = ["uart_parity_test.c"],
+    cw310 = new_cw310_params(
+        test_cmd = " ".join([
+            "--bootstrap=\"{firmware}\"",
+            "--firmware-elf=\"{firmware:elf}\"",
+        ]),
+        test_harness = "//sw/host/tests/chip/uart:uart_parity",
+    ),
+    exec_env = {
+        "//hw/top_earlgrey:fpga_cw310_sival": None,
+        "//hw/top_earlgrey:fpga_cw310_sival_rom_ext": None,
```

Comment:
I think that we should add the cw340 if the exec_env for it is merged before this one.